### PR TITLE
Update AUTHORS.md

### PR DESCRIPTION
**Pull request by EvgeniaChroni**
### Checklist:

- [x] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [x] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging` and not `master`.

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -56,6 +56,9 @@ To contributors: please add your name to the list when you submit a patch to the
    * SAR PySpark improvement
 * **[Daniel Schneider](https://github.com/danielsc)**
    * FastAI notebook
+* <b>Evgenia Chroni, Alexandros Ioannou, Kaisar Mussalim</b>
```
Comment (miguelgfierro) [inline AUTHORS.md:59]: can you guys please follow the same format as the other authors?

**Comment by miguelgfierro** [comment]
also, please send the PR to staging

File: AUTHORS.md (hunk 1 above)
Comment (EvgeniaChroni) [reply AUTHORS.md:59]: Yes sure. Given that we worked as a team, do you want to add our names separately or it is ok to add our names all together in one line ?

Comment (miguelgfierro) [reply AUTHORS.md:59]: Yeah, please add your names separately. Other people, even though as well worked together, had one entry with their name in alphabetical order.

**Comment by miguelgfierro** [comment]
already in the repo: https://github.com/microsoft/recommenders/pull/1194

**Pull request closed without merging**